docs/testing.rst

```diff
@@ -162,26 +162,34 @@ the real service is not configured).
 Calling ``stub_action``
 ***********************
 
-``stub_action`` has four potential arguments. Only the first two are required:
+``stub_action`` has five potential arguments. Only the first two are required:
 
 - ``service``: The name of the service on which this action will be called
 - ``action``: The name of the action to stub
 - ``body``: A dictionary containing the response body that the stubbed action should return (the same schema that
   would normally be returned from the action class's ``run`` method).
 - ``errors``: A list of SOA errors that should be raised, where each error is a dict with at least ``code`` and
   ``message`` keys and optionally a ``field`` for field errors.
+- ``side_effect``: A function, an exception class or instance, or an iterable. It behaves exactly like
```

Comment:
Please add:

>While it's possible to use `body` or `errors` in conjunction with `side_effect`, it is not recommended unless you really know what you're doing. It can be confusing for future developers working in your code. See the Python `unittest.Mock` documentation for a detailed description of the expected behavior when specifying `return_value` and `side_effect` at the same time.